tests/functional/object/mcg/test_custom_credentials_using_mcg_cli.py

```diff
@@ -0,0 +1,112 @@
+import logging
+import string
+import random
+
+from ocs_ci.framework.testlib import (
+    skipif_ocs_version,
+    MCGTest,
+    tier1,
+    red_squad,
+    mcg,
+)
+from ocs_ci.ocs import constants
+from ocs_ci.helpers.helpers import retrieve_cli_binary
+from ocs_ci.utility.utils import run_cmd, get_random_str
+from ocs_ci.helpers.helpers import get_s3_credentials_from_secret
+from pathlib import Path
+
+logger = logging.getLogger(__name__)
+mcg_cli = constants.NOOBAA_OPERATOR_LOCAL_CLI_PATH
+
+
+@tier1
+@red_squad
+@mcg
+class TestCustomCredsUsingMcgCli(MCGTest):
+    def update_nb_account(self, account_name, access_key, secret_key):
+        "Update noobaa account with custom credential values"
```

Comment:
Keep the Docstring format consistent across the file:
```suggestion
        """
        Update noobaa account with custom credential values
        """
```

Reply:
Done